Plugins/NativeServerPlugin/inc/buffer_renderer.h

```diff
@@ -0,0 +1,48 @@
+﻿#pragma once
+
+#include <wrl\client.h>
+#include <wrl\wrappers\corewrappers.h>
+#include <d3d11_4.h>
+#include <functional>
+
+#include "plugindefs.h"
+#include "webrtc/modules/video_coding/codecs/h264/include/nvEncodeAPI.h"
+#include "webrtc/modules/video_coding/codecs/h264/include/nvCPUOPSys.h"
+
+#include "webrtc/modules/video_coding/codecs/h264/h264_encoder_impl.h"
+
+namespace StreamingToolkit
+{
+    class BufferRenderer
```

Comment:
nit: can you add a brief comment describing what this component is for?

Reply:
Sure thing, totally forget to add the description for one of the most important class.